### ISSUE TITLE
Calling setEnvVar without converting response value to string results in crash when configuring environment

### ISSUE THREAD

**Issue by bjornevik** [usebruno/bruno#847]
**Version:** 1.0.0

A "crash" seems to be caused when using `setEnvVar` to set an environment variable to a response value which is not a string. This only seems to happen when setting equal to a value from the response. If using `setEnvVar("value", 123456789)` directly it works fine, but if `123456789` is part of the response-object it seems to not handle it correctly.

## Example
Given the following response from an API:
```json
{
  "value": 123456789,
}
```

The following script will cause a crash when clicking to configure environment:
```ts
const body = res.getBody()
bru.setEnvVar("value", body.value);
```

The following script will fill in the value as expected:
```ts
const body = res.getBody();
bru.setEnvVar("value", `${body.value}`)
```

## Screenshots

With following postscript:
![CleanShot 2023-11-01 at 13 14 08](https://github.com/usebruno/bruno/assets/5864000/05db2e24-17d7-4217-9b88-02409c137e81)

After running query:
![CleanShot 2023-11-01 at 13 18 28](https://github.com/usebruno/bruno/assets/5864000/f23e9109-97a5-4e0e-8914-7f5cf82ede05)

Results in:
![CleanShot 2023-11-01 at 13 15 11](https://github.com/usebruno/bruno/assets/5864000/3ea53476-b74e-4d3f-ada0-c0d662ab4d98)

**Comment by helloanoop** [comment]
@bjornevik Usually strings are expected to be store in the`setEnvVar`.
We should be able to fix this by doing a `JSON.stringify()` before saving and `JSON.parse()` before fetching

I need to think about this to see if there will be any effects.

Meanwhile, Can you use `bru.setVar('key', val)` and `bru.getVar('name')`
This allows storing full objects as well as exploring them in the Variable Viewer

![image](https://github.com/usebruno/bruno/assets/9350733/20cef03b-27d7-44bb-bb61-1d2afbb2e961)

**Comment by bjornevik** [comment]
Seems like this was fixed in #1285
